community/glorytun/depends

```diff
@@ -0,0 +1,2 @@
+pkgconf
```

Comment:
This should be: `pkgconf make` as I doubt it's needed at runtime.

Reply:
Thanks! I updated the PR accordingly :)